mu-plugins/blocks/query-filter/src/style.pcss

```diff
@@ -112,9 +112,26 @@
 	color: var(--wp--custom--wporg-query-filters--color--text);
 	display: none;
 
-	& .wporg-query-filter__modal-header {
+	&:where(:not(.is-single-select)) .wporg-query-filter__modal-header {
 		display: none;
 	}
+
+	& .wporg-query-filter__modal-header {
+		padding-block-start: var(--wp--preset--spacing--10);
```

Comment:
Can we make the padding match the left side:

| Current | Proposal |
|--------|--------|
| <img width="329" alt="Screenshot 2024-04-04 at 9 56 02 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/1657336/fa90dde5-d1e8-4071-92f3-2751dc86e6fa"> | <img width="321" alt="Screenshot 2024-04-04 at 9 54 38 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/1657336/0c978cc4-aaa4-439b-b0d8-792a8db18560">  |

Reply:
Looks like that actually should be 18px by the design, so I'll get that updated 👍🏻

Reply:
Updated the padding in https://github.com/WordPress/wporg-mu-plugins/pull/596/commits/eaa32e0e5ba3b9e54c4a35c25a2d0d6b6d1640b1